webpack.main.config.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
         { from: 'source/main/modules/export/assets/export.tpl.htm', to: 'assets' },
         { from: 'source/main/modules/export/assets/template.revealjs.htm', to: 'assets' },
         { from: 'source/main/modules/export/assets/revealjs-styles', to: 'assets/revealjs-styles' },
-        { from: 'source/main/modules/window-manager/assets/icons', to: 'assets/icons' }
+        { from: 'source/main/modules/window-manager/assets/icons', to: 'assets/icons' },
+        { from: 'resources/icons', to: 'assets/icons' }
```

Comment:
There are a lot of not-needed files in `resources/icons`. I don't think the Zettlr developers would like increasing the webpack size needlessly.
Instead, only copy in the files needed. I suggest:
```typescript
{ from: 'resources/icons/png', to: 'assets/icons/png' }
```